lib/victor/cli/commands/generate_svg.rb

```diff
@@ -0,0 +1,57 @@
+module Victor
+  module CLI
+    module Commands
+      class GenerateSVG < Base
+        summary "Convert Ruby code to SVG"
+
+        usage "victor to-svg RUBY_FILE [SVG_FILE --template TEMPLATE]"
+        usage "victor to-svg (-h|--help)"
+
+        option '-t, --template TEMPLATE', "Set SVG template\n"+
+          "Can be: default, html, minimal or a file path"
+
+        param "RUBY_FILE", "Input Ruby file"
+        param "SVG_FILE", "Output SVG file. Leave empty to write to stdout"
+
+        example "victor to-svg input.rb output.svg"
+        example "victor to-svg input.rb --template minimal"
+
+        def run
+          ruby_file = args["RUBY_FILE"]
+          svg_file = args["SVG_FILE"]
+
+          svg = setup_svg args['--template']
+          eval File.read(ruby_file)
```

Comment:
Does running `eval` here mean that the ruby_file has access to variables defined here? For example args and ruby_file, svg_file etc? Wondering if it should be run in a more contained environment. Thinking is that spelling mistakes or just randomly you will get access to things you have not defined.

Reply:
Also what happens to `require` or `require_relative` commands with eval? Would it still work as expected?

Reply:
Running `eval` is not ideal, I tried to avoid it using `load`, but then it does not have access to the `svg` object. Do you have a better idea to allow us to drop `eval`?